api/src/index.ts

```diff
@@ -1,7 +1,7 @@
-import app from './app';
+import {app} from './app'
```

Comment:
same comment as earlier, don't really see the need to stray from convention here.